src/api/auth.ts

```diff
@@ -14,6 +14,7 @@ type LoginParams = {
   mobile?: boolean;
   tenantId?: string;
   domainName?: string;
+  headers?: Record<string, any>;
```

Comment:
```suggestion
  headers?: Record<string, string>;
```

do we expect something else or do you think this would be okay ?